pipelines/jobs/pipeline_job_template.groovy

```diff
@@ -8,6 +8,7 @@ gitRefSpec = ""
 propagateFailures = false
 runTests = true
 runInstaller = true
+runSinger = false
```

Comment:
Please help me verify if it's ok to set false as default and where to activate signer.

Reply:
This should probably be `true` by default. Like `runInstaller`, `runSigner` should also be placed around Line 19 (inside the `if (binding.hasVariable('PR_BUILDER')) {` block and set to `false`.